app/models/game.rb

```diff
@@ -19,10 +19,15 @@ def render_piece(x, y)
 
   # Populate a game with all the pieces in the correct locations (x_coordinate, y_coordinate)
   def populate_game!
-    add_starting_pieces_for_color!('white')
-    add_starting_pieces_for_color!('black')
-    save
-    self
+    # if there are already pieces in this game,
+    # we don't want the game to be populated again.
+    if pieces.count > 0
```

Comment:
Can do this as a guard statement on first line of method

`return unless pieces.count == 0`